css/components/_btn.scss

```diff
@@ -0,0 +1,77 @@
+
+.btn {
+  border: 1px solid transparent;
+  border-radius: .25rem;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 400;
+  padding: 1px;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  vertical-align: middle;
+
+  a {
```

Comment:
The `a` inside a button is used at the moment for the `page preview` links - for when you click on a button and you want it to take you to a page. I could move this to an `on.click` event which may be cleaner?